client/src/partials/patients/record/patient_record.html

```diff
@@ -64,13 +64,17 @@
 
               <div class="col-sm-4">
                 <div style="border : 1px solid #ccc; height : 200px;">
-                  <!-- no image placeholder -->
-                  <div ng-if="!PatientRecordCtrl.patient.image" style="text-align : center;">
+                  <div ng-if="!PatientRecordCtrl.patient.avatar" style="text-align : center;">
                     <span style="font-size : 3em; color : #ccc; padding-top: 70px;" class="glyphicon glyphicon-picture"></span>
                   </div>
+                  <div ng-if="PatientRecordCtrl.patient.avatar" style="text-align : center;">
+                    <img ng-src="{{ PatientRecordCtrl.patient.avatar }}" width="120"> 
+                  </div>
                 </div>
 
-                <a class="btn btn-default btn-block" disabled><span class="glyphicon glyphicon-upload"></span> {{ "PATIENT_RECORDS.UPLOAD_PICTURE" | translate }}</a>
+                <button class="btn btn-default btn-block" type="file" ngf-select="PatientRecordCtrl.uploadFiles($file, $invalidFiles)" accept="image/*" ngf-max-height="1000" ngf-max-size="2MB">   <span class="glyphicon glyphicon-upload"></span> {{ PatientRecordCtrl.textButton }} 
```

Comment:
What happens if you change this `<input type="file>` instead of `<button>`?  Does it destroy the CSS?